x-pack/legacy/plugins/snapshot_restore/common/types/app.ts

```diff
@@ -5,7 +5,9 @@
  */
 
 export interface AppPermissions {
```

Comment:
Nit: should this be `AppPrivileges` or `AppAuthorization`?

Reply:
Good point, I'll change it to `AppAuthorization`.